packaging/python/test_requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest
+setuptools;python_version>='3.12' # From 3.12, no longer installed by default
+numpy
```

Comment:
What's the purpose of this file? List all the dependencies that the tests need but are not installed automatically by installing NEURON? If yes, why is `numpy` here?

Reply:
Yes, that is the purpose. This started out in the `test_wheels.sh` file and I moved it here. NumPy was in there explicitly and I didn't think to change it.

I removed it and it seems to work fine without. (We pip install the wheel after all.) Presumably it was in there because of the pin, or out of paranoia.